cmd/satellite/main.go

```diff
@@ -43,23 +44,35 @@ func main() {
 		hndlr = gpustats.NvidiaGPUHandler{}
 	}
 
+	// Send initial infopacket of GPUInfo
+	s.sendGPUInfo(hndlr)
+
+	// Set up neat ctrl-c breaking
+	termsig := make(chan os.Signal, 1)
```

Comment:
Why are we making this change here?!?

This should be a seperate PR

Reply:
Got carried away. Just a small thing that I wanted, but can remove if you want to.

Reply:
Yeah, let's please deal with this seperatly.